bundles/CoreBundle/Resources/translations/pl.extended.json

```diff
@@ -715,5 +715,6 @@
     "invitation_link_sent": "A temporary login link has been sent to email address: \"%s\" ",
     "geopolyline": "Geographic Polyline",
     "classificationstore_error_group_exists_msg": "Group with this name already exists",
-    "classificationstore_group_by_key": "Group by key"
+    "classificationstore_group_by_key": "Group by key",
+    "allow_to_create_new_object":  "Allow to create a new object in relation field"
```

Comment:
Please add the translation key only to the `en.extended.json`, all other languages are covered by POEditor.

Reply:
Unfortunately the translations for `de` are still there, can you please remove them as well. Thanks a lot!